### ISSUE TITLE
Error: 'bytes' object has no attribute 'format'

### ISSUE THREAD

**Issue by mikelove** [COMBINE-lab/FastaDigest#1]
hi Rob,

I did `pip3 install -e fasta_digest` and then I tried it out:

```
> compute_fasta_digest -h  
usage: compute_fasta_digest [-h] [--reference REFERENCE] [--out [OUT]]

Compute the signature of a reference fasta

optional arguments:
  -h, --help            show this help message and exit
  --reference REFERENCE
                        The reference fasta file
  --out [OUT]           The json file where output should be written

> compute_fasta_digest --reference gencode.v26.transcripts.fa --out digest 
Traceback (most recent call last):
  File "/usr/local/bin/compute_fasta_digest", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/Users/love/Downloads/FastaDigest-master/fasta_digest/bin/compute_fasta_digest", line 101, in <module>
    main(args)
  File "/Users/love/Downloads/FastaDigest-master/fasta_digest/bin/compute_fasta_digest", line 78, in main
    seq_hasher.update(b"{}".format(seq))
AttributeError: 'bytes' object has no attribute 'format'
```

this is on Mac with python3 installed via homebrew if that is useful information

```
> python3 --version
Python 3.6.2
> pip3 --version 
pip 9.0.1 from /usr/local/lib/python3.6/site-packages (python 3.6)
```

**Comment by rob-p** [comment]
Ahhhh!  It's a python 2 vs python 3 issue :(.  I bet this would work in a Python 2 environment.  I'll try and make the relevant changes to make it py3 compatible.

**Comment by mikelove** [comment]
i'm gonna tell everyone that Rob told me to use Python 2.6 instead of Python 3

**Comment by rob-p** [comment]
No, no, no .... definitely 2.7 ;-P.  I'll ping back here when I figure out this `byte` nonsense.

**Comment by rob-p** [comment]
Ok; should be fixed if you upgrade to `0.1.2` via pip.
